memory/memory_store.go

```diff
@@ -170,7 +171,7 @@ func (store *MemoryStore) Delete(id jstore.EntityID) error {
 	}
 
 	item, ok := store.storage[id.Project][id.DocumentType][id.ID]
-	if ok && (item.entity.Version != id.Version && id.Version != jstore.NoVersion) {
+	if ok && (item.entity.Version != id.Version && id.Version != nil) {
```

Comment:
Das zerbricht das Interface! In store.go sollte es dann `var NoVersion = nil` geben.

Reply:
Eine `var NoVersion = nil` ist nicht möglich. Ich habe mir mal angeguckt, was der Weg in Go ist um zu gucken, ob ein Interface nil ist und fand dies:
https://stackoverflow.com/questions/13476349/check-for-nil-and-nil-interface-in-go

Dementsprechend gibt es jetzt eine Methode auf EntityID.